lib/node_modules/@stdlib/blas/base/drot/docs/repl.txt

```diff
@@ -0,0 +1,134 @@
+
+{{alias}}( N, x, strideX, y, strideY, c, s )
+    Applies a plane rotation.
+
+    The `N` and stride parameters determine how values in the strided arrays are
+    accessed at runtime.
+
+    Indexing is relative to the first index. To introduce an offset, use typed
+    array views.
+
+    If `N` is less than or equal to `0`, the vectors are unchanged.
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    x: Float64Array
+        First input array.
+
+    strideX: integer
+        Index increment for `x`.
+
+    y: Float64Array
+        Second input array.
+
+    strideY: integer
+        Index increment for `y`.
+
+    c: number
+        Cosine of the angle of rotation.
+
+    s: number
+        Sine of the angle of rotation.
+
+    Returns
+    -------
+    y: Float64Array
+        Second input array.
+
+    Examples
+    --------
+    // Standard Usage:
+    > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+    > var y = new {{alias:@stdlib/array/float64}}( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+    > {{alias}}( x.length, x, 1, y, 1, 0.8, 0.6 )
```

Comment:
```suggestion
    > {{alias}}( x.length, x, 1, y, 1, 0.8, 0.6 );
```

For silencing REPL output, make sure that line which do not display expected results end in a semicolon, here and below.